community/contributing/best_practices_for_engine_contributors.rst

```diff
@@ -137,8 +132,8 @@ complexity to the codebase.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 For programmers, it is always a most enjoyable challenge to find the most
-optimal solutions to problems. Things, however, may go overboard sometimes and
-programmers will try to come up with solutions that solve as many problems as
+optimal solutions to problems. It is possible overboard, though. Sometimes,
```

Comment:
Missing a verb?

Reply:
Yep. I just force pushed a new version that adds the missing verb.